bedrock/firefox/templates/firefox/releases/aurora-notes.html

```diff
@@ -4,16 +4,31 @@
 
 {% extends "firefox/releases/release-notes.html" %}
 
-{% block body_class %}fx-notes space{% endblock %}
+{% block page_css %}
+  {% stylesheet 'firefox_releasenotes_firefox' %}
+{% endblock %}
 
 {% set channel_name = 'Aurora' %}
 
-{% block site_header_logo %}
-    <h2><a href="{{ url('firefox') }}">{{ high_res_img('firefox/template/header-logo-inverse.png', {'alt': 'Mozilla Firefox', 'width': '185', 'height': '70'}) }}</a></h2>
-{% endblock %}
-
 {% block platform_switch %}
-    {% if release.major_version()|int() < 35 %}
-	{{ super() }}
-    {% endif %}
+  <div id="nav" class="navigator">
+    <div class="container">
+      <ul class="menu">
+        <li><a href="{{ url('firefox') + 'notes/' }}">{{ _('Desktop') }}</a></li>
```

Comment:
@pmclanahan or @jgmize Is there a better way to reference these links? I'm not sure if there's a way within the `url()` function to references the various types of release notes pages.